utils/testConversation.py

```diff
@@ -125,8 +125,13 @@ async def fill_df(utterance, row_idx, out_df, workspace_id, conversation, apiver
             response_text_list = []
             if 'text' in resp['output']:
                 response_text_list = resp['output']['text']
-            elif 'generic' in resp['output'] and len(resp['output']['generic']) != 0:
+            elif 'generic' in resp['output'] and len(resp['output']['generic']) != 0 and 'text' in resp['output']['generic'][0]:
                 response_text_list = [ resp['output']['generic'][0]['text'] ]
+            elif 'generic' in resp['output'] and len(resp['output']['generic']) != 0 and 'title' in resp['output']['generic'][0]:
```

Comment:
@victorpovar can you add a comment here in the code that this block is supporting disambiguation nodes?
Otherwise looks good to me and nice catch.

Reply:
@andrewrfreed Done, added a comment. Thank you for the review.